promptsource/templates/GEM/totto/templates.yaml

```diff
@@ -0,0 +1,273 @@
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Guess the URL from which this table has been extracted!
+
+      |||
+
+      {{table_webpage_url}}
+
+      {% endif %}
+
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table webpage url
+    reference: ''
+  484004fa-baf7-4757-a796-aba9f8c887a8: !Template
+    answer_choices: null
+    id: 484004fa-baf7-4757-a796-aba9f8c887a8
+    jinja: "{% if target != \"\" %}\nTable Page Title: {{table_page_title}} \nTable\
+      \ Section : {{table_section_title}}\n\nHow can you best use the above information\
+      \ and convert the  below highlighted cells of a table to a sentence?\n\nHighlighted\
+      \ Cells:\n{% for cell in highlighted_cells: %}\n{% if table[0]|length >= cell[1]\
+      \ and table[0]|length >= cell[0] %}\n{{table[0][cell[1]][\"value\"]}}: {{table[cell[0]][cell[1]][\"\
+      value\"]}}{% endif %}{% endfor %}\n|||\n{{target}}\n{% endif %}\n"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: best_describe_highlighted_sentence
+    reference: ''
+  4ef9e76e-7c03-4869-abdc-40ace4ce4c1c: !Template
+    answer_choices: null
+    id: 4ef9e76e-7c03-4869-abdc-40ace4ce4c1c
+    jinja: "{% if target != \"\" %}\nGiven the following information about a table\
+      \ and summarize the information from the highlighted cell:\nTable Page Title:\
+      \ {{table_page_title}} \nTable Section : {{table_section_title}}\"\nValues from\
+      \ Highlighted Cells: \n{% for cell in highlighted_cells: %}\n{% if table[0]|length\
+      \ >= cell[1] and table[0]|length >= cell[0] %}\n{{table[0][cell[1]][\"value\"\
+      ]}}: {{table[cell[0]][cell[1]][\"value\"]}}{% endif %}{% endfor %}\n|||\n{{target}}\n\
+      {% endif %}\n"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: summarize_the_highlighted_cell
+    reference: ''
+  7a2fb34a-c792-41b3-9682-fcda15f1d100: !Template
+    answer_choices: null
+    id: 7a2fb34a-c792-41b3-9682-fcda15f1d100
+    jinja: '{% if table_section_title != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section title?
+
+      |||
+
+      {{table_section_title}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
```

Comment:
since we are currently not evaluating on `original_task: false`, for the sake of simplicity, could we exclude these non-original tasks?